python/GafferUI/SpreadsheetUI/_PlugTableView.py

```diff
@@ -352,12 +357,116 @@ def __plugMetadataChanged( self, nodeTypeId, plugPath, key, plug ) :
 
 				self.__applyColumnVisibility()
 
+	def __dragEnter( self, widget, event ) :
+
+		if Gaffer.MetadataAlgo.readOnly( self._qtWidget().model().rowsPlug() ) :
+			return False
+
+		if not isinstance( event.data, Gaffer.Plug ) :
+			return False
+
+		self.__lastDragPlug = None
+		return True
+
+	def __dragMove( self, widget, event ) :
+
+		pos = self.__positionInCellGrid( event )
+		cellPlug = self.plugAt( pos )
+
+		if self.__lastDragPlug == cellPlug or ( cellPlug is not None and cellPlug.isSame( self.__lastDragPlug ) ) :
```

Comment:
Been burnt by getting different python wrappers for plugs before now. `cellPlug` comes from a subscript into a `RowPlug`.

Reply:
I think you were burnt by `is`, not `==`. We've [implemented `==` specifically](https://github.com/ImageEngine/cortex/commit/b13d057905492fa72cdd6e75605920ec96fd0fe5#diff-967aada24169dcc8c1e55658954456cadbfcb4f1dcb3dc6ed451118a2e987f59) so it can be used as I suggested :

```
> p1 = root['Plane']['name']
> p2 = root['Plane']['name']
> p1 is p2 # Aargh, burning!
False
> p1 == p2 # Ah, soothing
True
```

Reply:
Ahh splendid, thanks. A :fire: free version [here](https://github.com/GafferHQ/gaffer/pull/4045/commits/2262c5c3735cccfd0bcd96d149c6e5be2fb48769).